web/webpack.config.js

```diff
@@ -48,7 +47,8 @@ module.exports = function (env) {
         tty: require.resolve("tty-browserify")
       },
       alias: {
-        "@src": path.resolve(__dirname, "src")
+        "@src": path.resolve(__dirname, "src"),
+        "handlebars" : "handlebars/dist/handlebars.js"
```

Comment:
My understanding is that this is forcing `from "handlebars"` to resolve to the module's `/dist/handlebars.js`

Reply:
I believe so yeah, was the only thing that could get it to work...for the life of my everything else I was trying RE importing it in the component either wouldn't resolve it or would have compiler errors.